lib/node_modules/@stdlib/stats/base/nanmin/test/test.nanmin.js

```diff
@@ -21,7 +21,7 @@
 // MODULES //
 
 var tape = require( 'tape' );
-var floor = require( '@stdlib/math/base/special/floor' );
+var toAccessorArray = require( '@stdlib/array/base/to-accessor-array' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var isNegativeZero = require( '@stdlib/math/base/assert/is-negative-zero' );
 var Float64Array = require( '@stdlib/array/float64' );
```

Comment:
L28 will need to be updated after renaming `nanmin.js`. May apply elsewhere, as well.